test-e2e/project-crud.js

```diff
@@ -124,6 +124,37 @@ test('CRUD operations', async (t) => {
         'expected values returns from getMany()'
       )
     })
+    t.test('create, close and then create, update', async (st) => {
+      const projectId = await manager.createProject()
+      const project = await manager.getProject(projectId)
+      const values = new Array(5).fill(null).map(() => {
+        return getUpdateFixture(value)
+      })
+      for (const value of values) {
+        // @ts-ignore
+        await project[schemaName].create(value)
+      }
+      // @ts-ignore
+      const written = await project[schemaName].create(value)
+      await project.close()
+
+      await st.exception(async () => {
+        const updateValue = getUpdateFixture(value)
+        // @ts-ignore
+        await project[schemaName].update(written.versionId, updateValue)
+      }, 'should fail updating since the project is already closed')
+
+      await st.exception(async () => {
+        for (const value of values) {
+          // @ts-ignore
+          await project[schemaName].create(value)
+        }
+      }, 'should fail creating since the project is already closed')
+
+      await st.exception(async () => {
```

Comment:
trying to see why this test fails since we expect an error here (the db should be close by now and tests show that). I don't know why it actually fails, maybe it doesn't throw an `Error` ? @achou11

Reply:
yeah this is specific to brittle, which will fail the assertion since it's a native error that is raised in this case: https://github.com/holepunchto/brittle/tree/5eda53924e146ce99c3ed9f8770703c2c369df64#texceptionpromisefunctionasync-function-error-message

here's an updated version of the assertion that passes:

```js
await st.exception(async () => {
  try {
    await project[schemaName].getMany()
  } catch (err) {
    // A TypeError is originally raised but Brittle fails this assertion for native errors, including the TypeError that's originally raised here
    // https://github.com/holepunchto/brittle/tree/5eda53924e146ce99c3ed9f8770703c2c369df64#texceptionpromisefunctionasync-function-error-message
    throw new Error(err)
  }
}, 'should fail updating since the project is already closed')
 ```